src/pages/Filter/Filter.scss

```
@@ -0,0 +1,128 @@
+@import '../../styles/common.scss';
+
+.FilterBox {
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  margin-left: 0;
+  margin-right: 40px;
+  padding-left: 20px;
+  border-color: $color-grey;
+
+  .searchFilter {
+    .selectHead {
+      display: flex;
+      justify-content: space-between;
+
+      .title {
+        margin-left: 15px;
+        margin-top: 10px;
+        padding: 10px;
+      }
+    }
+
+    .selectRating {
+      display: flex;
+      justify-content: flex-start;
+      padding-top: 5px;
+      padding-bottom: 5px;
+      padding-left: 30px;
+      border-color: $color-grey;
+
+      .selectList {
+        display: inline-block;
+        text-align: left;
+        width: 100px;
+        margin: 5px;
+        margin-left: 5px;
+        padding: 5px;
+        outline: 0;
+        background-color: transparent;
+        border: 0px transparent solid;
+
+        &.on {
+          color: $color-main;
+        }
+      }
+    }
+  }
+
+  .priceForOnePerson {
+    .selectHead {
+      display: flex;
+      justify-content: flex-start;
+      padding: 5px;
+
+      .title {
+        margin-left: 20px;
+        margin-top: 10px;
+      }
+    }
+
+    .priceSelectSection {
+      padding-left: 30px;
+      .selectPriceList {
+        display: inline-block;
```

```suggestion
    .priceSelectSection {
      padding-left: 30px;
      
      .selectPriceList {
        display: inline-block;
```
새로운 셀렉터 앞에서 한줄 띄워주세요